images/nginx-drupal/drupal/favicon.conf

```diff
@@ -0,0 +1,12 @@
+## Support for favicon. Return an 1x1 transparent GIF if it doesn't
+## exist.
+location = /favicon.ico {
+  expires 30d;
+  try_files /favicon.ico @empty;
+}
+
+## Return an in memory 1x1 transparent GIF.
```

Comment:
let's leave @empty in the default config file, I don't think we need that to be easy overwrite-able and instead let's just include the `location = /favicon.ico` inside `drupal/favicon.conf`

Reply:
Good point, we might want `@empty` to exist for other places as well - defining that centrally is a better solution.